api/v1alpha1/virtualmachineclass_types.go

```diff
@@ -8,10 +8,29 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+// VGPUDevice contains the configuration corresponding to a vGPU device.
+type VGPUDevice struct {
+	ProfileName string `json:"profileName,omitempty"`
```

Comment:
Being the only field, I don't think `omitempty` make sense

Reply:
You're right. I removed them now.